src/state/token-lists/reducer.ts

```diff
@@ -63,15 +76,27 @@ export const getDefaultByUrl = () => ({
     version: { major: 0, minor: 0, patch: 0 },
     hasLoaded: false,
     requestId: '',
+    fetchable: true,
   },
 });
 export const initialState: TokenListsState = {
   activeLists: ['Mean Finance Graph Allowed Tokens'],
   activeAggregatorLists: [
     'https://raw.githubusercontent.com/Mean-Finance/token-list/main/mean-finance.tokenlist.json',
     'tokens.1inch.eth',
+    'custom-tokens',
   ],
   byUrl: getDefaultByUrl(),
+  customTokens: {
+    name: 'custom-tokens',
+    logoURI: '',
+    timestamp: new Date().getTime(),
```

Comment:
What is this timestamp for?

Reply:
it's the timestamp of the last updated value, it is from the schema from how you build official Token Lists